### PR TITLE
feat(whiteboard) add native implementation

### PR DESCRIPTION
**Pull request by mihhu**
static page deployed here: https://miih.jitsi.net/

### CODE REVIEW

File: css/main.scss [hunk 1]
```diff
@@ -36,6 +36,7 @@ $flagsImagePath: "../images/";
 @import 'modals/invite/info';
 @import 'modals/screen-share/share-audio';
 @import 'modals/screen-share/share-screen-warning';
+@import 'modals/whiteboard';
```
Comment (saghul) [inline css/main.scss:39]: Oh, can't we really not add more stuff here?

File: react/features/whiteboard/components/native/Whiteboard.tsx [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+     * immediately after mounting occurs.
+     *
+     * @inheritdoc
+     * @returns {void}
+     */
+    componentDidMount() {
+        const { navigation, t } = this.props;
+
+        navigation.setOptions({
+            headerTitle: t('whiteboard.screenTitle')
```
Comment (saghul) [inline react/features/whiteboard/components/native/Whiteboard.tsx:90]: Isn't this set in the screen definition?

**Review by saghul — APPROVED** [review]
Impressive work! Minor questions, but can be addressed later, let's get this out and test it! 🚀

File: css/main.scss (hunk 1 above)
Comment (mihhu) [reply css/main.scss:39]: Oh sorry, should I move the style into the _base file or simply have a style object passed directly?

**Pull request merged**